corehq/apps/users/models.py

```diff
@@ -802,9 +802,25 @@ class LastSubmission(DocumentSchema):
     commcare_version = StringProperty()
 
 
-class ReportingMetadata(DocumentSchema):
+class LastSync(DocumentSchema):
+    sync_date = DateTimeProperty()
+    build_version = IntegerProperty()
+    app_id = StringProperty()
 
+
+class LastBuild(DocumentSchema):
+    build_version = IntegerProperty()
+    build_version_date = DateTimeProperty()
+    app_id = StringProperty()
+
+
+class ReportingMetadata(DocumentSchema):
     last_submissions = SchemaListProperty(LastSubmission)
+    last_submission_for_user = SchemaProperty(LastSubmission)
+    last_syncs = SchemaListProperty(LastSync)
+    last_sync_for_user = SchemaProperty(LastSync)
+    last_builds = SchemaListProperty(LastBuild)
+    last_build = SchemaProperty(LastBuild)
```

Comment:
I know I already approved this on the other PR but just thinking that these names are a bit confusing and also inconsistent e.g. `last_build` should be `last_build_for_user` to be consistent with the other 2.

To make it very clear we could call them `last_X_across_apps`. 

At this point I'd also be OK with a comment or two and maybe grouping them into the global ones and the per app ones.